seed/challenges/01-front-end-development-certification/basic-bonfires.json

```diff
@@ -671,6 +671,7 @@
       "title": "Seek and Destroy",
       "description": [
         "You will be provided with an initial array (the first argument in the destroyer function), followed by one or more arguments. Remove all elements from the initial array that are of the same value as these arguments.",
+        "<strong>Note:</strong> You have to use the <code>arguments</code> object.",
```

Comment:
I think we usually have a `<br>` after the Note and no colon like in [this line](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/1f105f38610bd442fcad7ebabbc82406843ee5f6/seed/challenges/01-front-end-development-certification/basic-javascript.json#L505).